appveyor.yml

```diff
@@ -7,9 +7,10 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
-  - npm -g install npm@latest
+  - set AVA_APPVEYOR=true
+  - npm -g install npm@latest || (timeout 30 && npm -g install npm@latest)
   - set PATH=%APPDATA%\npm;%PATH%
-  - npm install
+  - npm install || (timeout 30 && npm install)
```

Comment:
I don't think it matters to be honest. AppVeyor is sooooo slow regardless. But we can try with 10 seconds to see if it still works.